src/Clerk.ts

```diff
@@ -50,18 +53,30 @@ export default class Clerk {
     serverApiUrl = defaultServerApiUrl,
     apiVersion = defaultApiVersion,
     httpOptions = {},
+    jwksCacheMaxAge = defaultJWKSCacheMaxAge,
   }: {
     apiKey?: string;
     serverApiUrl?: string;
     apiVersion?: string;
     httpOptions?: object;
+    jwksCacheMaxAge?: number;
   } = {}) {
     this._restClient = new RestClient(
       apiKey,
       serverApiUrl,
       apiVersion,
       httpOptions
     );
+
+    this._jwksClient = jwks({
+      jwksUri: `${serverApiUrl}/${apiVersion}/jwks`,
+      requestHeaders: {
+        Authorization: `Bearer ${apiKey}`,
+      },
+      timeout: 30000,
```

Comment:
🔧 The default timeout for this particular request in the Ruby SDK was set to 5 seconds as per Sokratis' comment. It's better to keep the SDKs as consistent as possible in their behavior, so I suggest to change this timeout as well.

https://github.com/clerkinc/clerk-sdk-ruby/blob/main/lib/clerk/resources/jwks.rb#L13

Reply:
Fixed